tests/examples/setuptools-with-imports-in-setup-py/dependencies.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+files:
+  py_rapids_build:
+    output: pyproject
+    pyproject_dir: .
+    extras:
+      table: tool.rapids-build-backend
+      key: requires
+    includes:
+      - build_python
+dependencies:
+  build_python:
+    specific:
+      - output_types: [pyproject, requirements]
+        matrices:
+          - matrix: {cuda: "85.*"}
+            packages:
+              - matplotlib
```

Comment:
Should we pick a smaller package like `requests` or `more-itertools`?

Reply:
Sure, I'll switch this to `more-itertools`.

I'd originally chosen `matplotlib` because I was looking for this mix:

* actively maintained
* wheels for many platforms / Python versions
* unlikely to already be in the `rapids-build-backend` CI environment

But since this PR has a test that'll fail in the "optional dependency was accidentally found because it was in the CI environment" case, I'm not as worried about that third condition.

Reply:
changed this to `more-itertools`